lib/ram_pwrdn/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config RAM_POWER_DOWN_LIBRARY
+	bool "Enable API for turning off unused RAM segments"
+	help
+	  This allows application to call API for disabling unused RAM segments.
+	  This is usually not needed, but can improve battery lifetime for
+	  applications that spend most of the time in the sleep mode with most
+	  peripherals disabled.
+
+# Configure RAM_POWERDOWN_LOG_LEVEL
+module =  RAM_POWERDOWN
+module-str = RAM Power down module
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
```

Comment:
Please place the log level setting under RAM Power down menu, as is done here:
https://github.com/nrfconnect/sdk-nrf/blob/master/lib/dk_buttons_and_leds/Kconfig#L11-L33

Reply:
Fixed.

Reply:
Not completely, you forgot to change: `config RAM_POWER_DOWN_LIBRARY`
https://github.com/nrfconnect/sdk-nrf/blob/0b69744149ad5eea8fd65b74e312d8afe63fb760/lib/ram_pwrdn/Kconfig#L7

into a menuconfig as here: https://github.com/nrfconnect/sdk-nrf/blob/master/lib/dk_buttons_and_leds/Kconfig#L7

Reply:
I missed that, thanks!
Fixed now.